src/__tests__/scss/__snapshots__/styles-test.js.snap

```diff
@@ -23428,7 +23396,9 @@ li.bx--accordion__item--disabled:last-of-type {
   transition: 240ms cubic-bezier(0.2, 0, 0.38, 0.9) background-color;
   margin-left: auto;
   color: var(--cds-inverse-01, #161616);
-  transition: 110ms background-color cubic-bezier(0.2, 0, 0.38, 0.9);
+  transition-duration: 110ms;
```

Comment:
Do the props need to be broken out like this now in order to pass the linter?

Reply:
That's right — [transition shorthands must conform as expected](https://github.com/carbon-design-system/stylelint-plugin-carbon-tokens/tree/master/src/rules/layout-token-use#layout-token-use), but individual properties are accepted.